pandas/io/excel/_base.py

```diff
@@ -289,6 +289,8 @@
 
     .. versionadded:: 2.0
 
+engine_kwargs : Arbitrary keyword arguments passed to excel engine
```

Comment:
This line should only have the type and default (if applicable). The line after has the description. Also add a `versionadded`.

Reply:
Note this comment also applies to other docstrings